gtk/src/drag_context.rs

```diff
@@ -24,6 +24,7 @@ pub trait DragContextExtManual: 'static {
     fn drag_set_icon_name(&self, icon_name: &str, hot_x: i32, hot_y: i32);
 
     #[doc(alias = "gtk_drag_set_icon_pixubf")]
+    #[doc(alias = "gtk_drag_set_icon_pixbuf")]
```

Comment:
hmm i can't tell the difference with the existing doc alias here?

Reply:
I can't either, I guess it's a bug. :)

Reply:
Oh I see it now! The `u` and `b` are not in the right place. That's pretty funny. :)

Reply:
lol